src/views/SystemProfile/components/ToolsAndSoftware/index.tsx

```diff
@@ -1,100 +1,138 @@
-import React from 'react';
+/* eslint-disable no-console */
```

Comment:
is this needed?

Reply:
nope - was doing some console debugging and forgot to remove this. good catch.